addons/wardrobe/functions/fnc_replace.sqf

```diff
@@ -104,4 +104,6 @@ if !(".paa" in _imgNotify) then {
     _duration * 1.2
 ] call CBA_fnc_waitAndExecute;
 
+[QGVAR(itemChanged), [_classOrigin, _classTarget], _player] call CBA_fnc_targetEvent;
```

Comment:
maybe add unit to args
it's going to be ace_player most of the time, but it shouldn't hurt it add

```suggestion
[QGVAR(itemChanged), [_player, _classOrigin, _classTarget], _player] call CBA_fnc_targetEvent;
```